source/guides/account/scams-and-security.rst

```diff
@@ -0,0 +1,104 @@
+
+*******************************************
+Sweeping with Aggregate Bonded Transactions
+*******************************************
+
+What is an Aggregate Bonded transaction?
+========================================
+
+Aggregate Bonded transactions can include a variety of elements such as multiple signers, messages, and transfers. New Symbol users may not be familiar with the complex nature of Aggregate Bonded transactions, and must take great care to fully understand the full scope of any transactions they are asked to sign. For more information see :doc:`Aggregate Transaction <../../concepts/aggregate-transaction>`
+
+
+What is sweeping?
+=================
+
+Sweeping involves a malicious script that monitors certain types of transactions, then initiates a malicious Aggregate Bonded transaction disguised to appear related to the targeted transaction (i.e. by including a relevant message), but which also include a transfer of XYM or other mosaics to the bad actor's wallet. Signing these transactions can completely empty a user's wallet, so please make sure you carefully review before signing any transactions that you did not initiate.
+
+It may not be immediately apparent that you've been hacked by a sweeper. If you assumed that the transaction you signed was part of a process you were completing, you may also assume that your balance has been reduced by a stuck transaction, pending activation of delegated harvesting, or that your balance was transferred to a delegated harvesting address linked to your account.
+
+What could a sweeping attack look like?
+========================================
+
+- You begin Delegated Harvesting activation by propagating the 'Link all keys' transaction to the network
+- The bad actor's script detects this transaction and sends you an Aggregate Bonded transaction. This transaction might include a message related to harvesting such as "Delegated-№234567" as well as a transfer of XYM to an address controlled by the bad actor. 
+- Your wallet sees the Aggregate Bonded transaction included in a subsequent block and prompts you with the option to review and sign it.
+- You choose to sign the transaction, and your XYM balance is reduced by the transaction amount (typically all or nearly all of your XYM).
+
+
+********
+Spoofing 
+********
+
+What is spoofing?
+==================
+
+Spoofing involves hiding or disguising identity to enable malicious activity, literally spoofing the identity of the malicious party to make it believable and appear trustworthy. 
+
+What could a spoofing attack look like?
+========================================
+
+A spoofing hack will target private key or  mnemonic phrase (also known as your secret recovery phrase or seed phrase), as this can be used to restore your wallet and will provide a hacker with access to your private keys and the wallet's contents. Symbol wallets are a non-custodial , meaning you are responsible for keeping your private keys and mnemonic phrase secure.
```

Comment:
```suggestion
A spoofing hack will target private key or mnemonic phrase (also known as your secret recovery phrase or seed phrase), as this can be used to restore your wallet and will provide a hacker with access to your private keys and the wallet's contents. Symbol wallets are a non-custodial, meaning you are responsible for keeping your private keys and mnemonic phrase secure.
```